package_test.go

```diff
@@ -595,10 +593,21 @@ func (s *PackageSuite) TestRun(c *C) {
 
 	db := sqlair.NewDB(sqldb)
 
+	var outcome = sqlair.Outcome{}
+
 	insertStmt := sqlair.MustPrepare("INSERT INTO person VALUES ( $Person.name, $Person.id, $Person.address_id, 'jimmy@email.com');", Person{})
-	err = db.Query(nil, insertStmt, &jim).Run()
+	err = db.Query(nil, insertStmt, &outcome, &jim).Run()
```

Comment:
This is mixing an output parameter together with the input parameters, so feels like a slightly odd place to hang it. Also, this makes it tempting to store the value in the Query, as is being currently done in the PR, but that doesn't seem entirely right either. The easiest way to see it is imaginging the same query executed multiple times, each with a different outcome. It'd be odd to have that same value reused over and over. Instead, I think we want to pair this with `One` and `Decode`, which are the places where the output parameters are provided, and these are already independent for each query.